packages/app/src/containers/Canvas.tsx

```diff
@@ -106,6 +108,17 @@ export default memo(function Canvas() {
   const containerSize = useSize(containerRef);
   const meta = useSelector(Selectors.getCurrentPageMetadata);
   const { setCanvasSize, highlightLayer, highlightedLayer } = useWorkspace();
+  const bind = useGesture({
+    onWheel: ({ movement: [x, y] }) => {
+      const rawPoint = { x, y };
+      dispatch('interaction', [
+        state.interactionState.type === 'maybePan'
```

Comment:
@dabbott this isn't working right now. I had something similar to what you have below, but that was interfering with the current panning approach. Should we add a specific case for trackpad panning in the reducer or should we be able to tweak `updatePanning` to work here?

Reply:
I'm leaning toward a separate action. The current panning is stateful: you have to be in panMode, and then go from maybePan to panning, and there are several actions to coordinate that. Panning via gesture should theoretically be stateless... i.e. it could all happen with a single `pan` action that moves the canvas by a delta. It wouldn't need to go through `interactionState`, since you can always pan via trackpad even when you're in another state.